### PR TITLE
Add a subcommand to run the first come first served sale 

### PR DESCRIPTION
**Pull request by darkturo**

### CODE REVIEW

File: borderland-tool/borderland_tool/pretix/__init__.py [hunk 1]
```diff
@@ -4,27 +4,30 @@
...
         self.event = event
         self.token = token
+        self.url_scheme = "https"
+        if no_ssl:
+            self.url_scheme = "http"
 
 
     def get_orders(self, status="p"):
-        url = "https://{}/api/v1/organizers/{}/events/{}/orders/?page=1".format(self.host, self.org, self.event)
+        url = f"{self.url_scheme}://{self.host}/api/v1/organizers/{self.org}/events/{self.event}/orders/?page=1"
```
Comment (aleks-delta) [inline borderland-tool/borderland_tool/pretix/__init__.py:18]: looks good

Comment (aleks-delta) [reply borderland-tool/borderland_tool/pretix/__init__.py:18]: perhaps factor out the part before until {self.event}, it repeats in many parts of the code. Would avoid one-off errors

File: borderland-tool/borderland_tool/fcfs/first_come_first_served.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+class FCFS:
+    def __init__(self, pretix, csvfile, quota):
+        self.pretix = pretix
+        self.csvfile = csvfile
+        self.quota = quota
+        self.registered_without_membership_csv = self.load_csv()
+        self.has_order = self.has_voucher = None
+
+    def registrations_to_csv(self):
```
Comment (aleks-delta) [inline borderland-tool/borderland_tool/fcfs/first_come_first_served.py:18]: also a lot of repetition here from the lottery class, but i guess in the short term it's safer.

**Review by aleks-delta — APPROVED** [review]

File: borderland-tool/borderland_tool/pretix/__init__.py (hunk 1 above)
Comment (darkturo) [reply borderland-tool/borderland_tool/pretix/__init__.py:18]: yep, that's something that we could do later

File: borderland-tool/borderland_tool/fcfs/first_come_first_served.py (hunk 2 above)
Comment (darkturo) [reply borderland-tool/borderland_tool/fcfs/first_come_first_served.py:18]: I'm aware, this started as a quick copy of the "lottery" command

**Pull request merged**